tests/nightly/test_large_array.py

```diff
@@ -1168,6 +1169,76 @@ def check_correctness(mxnet_op, numpy_op, atol=1e-3):
     check_binary_broadcast()
 
 
+def test_linalg():
+    def batchify(mat):
+        mat_np = mat.asnumpy()
+        return nd.array([mat_np, mat_np])
+
+    def check_diag(mat, val):
+        for i in range(LARGE_SQ_X):
+            assert mat[i, i] == val
+
+    def check_batch_diag(mat, val):
+        check_diag(mat[0], val)
+        check_diag(mat[1], val)
+
+    def run_det(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = det(inp)
+        return inp.grad, out
+
+    def run_inverse(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = inverse(inp)
+        return inp.grad, out
+
+    def run_trmm(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = trmm(inp, inp)
+        return inp.grad, out
+
+    def run_trsm(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = trsm(inp, inp)
```

Comment:
I don't see `trsm`/`inverse`/`det`/`trmm` or any of these ops being imported from `mx.nd.linalg` namespace.
How is it being accessed? Do these tests work? Can you please paste the results?

Reply:
You can run these tests with
```
pytest tests/nightly/test_large_array.py::<insert function_name>
```

Reply:
Added namespace